package/skelcd-control-suse-manager-server.spec

```diff
@@ -38,8 +38,8 @@ BuildRequires:  libxml2-tools
 BuildRequires:  yast2-installation-control >= 4.1.5
 
 # Original SLES control file (skip registration)
-BuildRequires: skelcd-control-SLES >= 15.2.0
-BuildRequires: diffutils
+BuildRequires:  diffutils
+BuildRequires:  skelcd-control-SLES >= 15.2.0
```

Comment:
Do I need to change this from `15.2.0` to `15.4.0`? Even changing it the CI failed with the same message.

Reply:
@lslezak can you check ci failure? it looks like in our repos it is missing some migration product.